project/config/afbini/config/node.type.article.yml

```diff
@@ -11,10 +11,10 @@ third_party_settings:
     parent: ''
   origins_toc:
     toc_enable: 1
-    toc_title: 'Table of contents'
-    toc_location: '#post-content .page-content'
+    toc_title: Contents
+    toc_location: .field-summary
     toc_insert: before
```

Comment:
I think this should be after

Reply:
Thats it updated, moved Toc to after summary field

Reply:
I would recommend changing that back to .field-summary and just changing the Insert dropdown to after